test/project_test.dart

```diff
@@ -9,57 +9,46 @@ library linter.test.project;
 import 'dart:io';
 
 import 'package:linter/src/project.dart';
-import 'package:unittest/unittest.dart';
+import 'package:test/test.dart';
 
-main() {
-  groupSep = ' | ';
-
-  defineTests();
-}
-
-defineTests() {
```

Comment:
pkg/test == new world
groupSep is also gone.

All that type of visual tweaking will happen in custom runners